desk/mar/chat/heads-2.hoon

```diff
@@ -0,0 +1,14 @@
+/-  c=chat
+/+  j=chat-json
+|_  =chat-heads:c
```

Comment:
This obviously works, but since we decided to version chat with separate sur files, this should ideally import the versioned sur file. Then we don't need to adjust all these marks next time :-)

```suggestion
/-  c=chat-5
/+  j=chat-json
|_  =chat-heads:c
```